inlong-agent/conf/agent.properties

```diff
@@ -112,14 +112,13 @@ agent.manager.auth.secretKey=
 
 
 ############################
-# prometheus config
+# metric config
 ############################
-# whether to enable prometheus
-agent.prometheus.enable=true
+metricDomains=Agent
```

Comment:
it's designed for `agent`, maybe the `metricDomains` param is needless?

Reply:
has been deleted